pkg/daemon/file_writers.go

```diff
@@ -45,12 +46,62 @@ func noOrigFileStampName(fpath string) string {
 	return filepath.Join(noOrigParentDir(), fpath+".mcdnoorig")
 }
 
+func isFileOwnedByRPMPkg(fpath string) (string, bool, error) {
```

Comment:
In both calls to this function, only the middle bool (isowned) is used. Was that how you intended to use this? If so, we can maybe just return the bool.

Also, I think we should always process the error. In the calls to this function the error is ignored. I think we should consider processing the error properly